docs/UserGuide.md

```diff
@@ -38,14 +38,14 @@ BookFace replaces a paper-based system or manual tracking of books, providing gr
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `BookFace.jar` from here.
+2. Download the latest `BookFace.jar` from here.
```

Comment:
The list gets automatically numbered, so it's easier to add new steps later. But we're not changing this anyways